packages/core/src/system.ts

```diff
@@ -71,11 +71,17 @@ export function createSystem<T extends ActorSystemInfo>(
 }
 export interface BaseInspectionEventProperties {
   rootId: string; // the session ID of the root
+  /**
+   * The relevant actorRef for the inspection event.
+   * - For snapshot events, this is the `actorRef` of the snapshot.
+   * - For event events, this is the target `actorRef` (recipient of event).
+   * - For actor events, this is the `actorRef` of the registered actor.
+   */
+  actorRef: AnyActorRef;
 }
 
 export interface InspectedSnapshotEvent extends BaseInspectionEventProperties {
   type: '@xstate.snapshot';
-  actorRef: AnyActorRef; // Only available locally
```

Comment:
note that by "duplicating" this property here you could have a JSDoc unique to this particular event type here